### ISSUE TITLE
hdWGCNA: Could't Find some functions in the workflow

### ISSUE THREAD

**Issue by RcuIT** [smorabit/scWGCNA#106]
**Describe the bug**
I Installed the hdWGCNA on my mac computer. I started to follow the hdWGCNA for scRNA seq tutorial using my own data set. I performed upto `ModuleConnectivity`function. When I started to perform `ResetModuleNames`, it give me the an error saying could not find function `GetROCData`.  When I check, I found I can't find it in hdWGCNA install on my mac computer but I can see the function in the hdWGCNA on my windows laptop. Any suggestion to proceed ?

```
# part of my code is here 
# variable features are assigned to a vector
var.features <- sobj_w.3@assays$RNA@var.features

sobj_w.3 <- SetupForWGCNA(sobj_w.3, features = var.features, wgcna_name = "week.3")
# construct metacells  in each group
sobj_w.3 <- MetacellsByGroups(
  seurat_obj = sobj_w.3,
  group.by = "cluster.names", 
  reduction = 'pca', 
  k = 25, 
  max_shared = 10, 
  ident.group = 'cluster.names' 
)
# normalize metacell expression matrix:
sobj_w.3 <- NormalizeMetacells(sobj_w.3)
sobj_w.3 <- SetDatExpr(
  sobj_w.3,
  group_name = "progenitor", 
  group.by="cluster.names",
  assay = 'RNA',
  slot = 'data' 
)
# Test different soft powers:
sobj_w.3 <- TestSoftPowers(
  sobj_w.3,
  networkType = 'signed' 

# plot the results
plot_list <- PlotSoftPowers(sobj_w.3)
# assemble with patchwork
wrap_plots(plot_list, ncol=2)

# construct co-expression network:
sobj_w.3 <- ConstructNetwork(
  sobj_w.3, soft_power=5,
  setDatExpr=FALSE,
  tom_name = 'Week.3.p' 
)
PlotDendrogram(sobj_w.3, main='Dendrogram')
TOM <- GetTOM(sobj_w.3)
# need to run ScaleData first or else harmony throws an error:
sobj_w.3 <- ScaleData(sobj_w.3, features=var.features)

# compute all MEs in the full single-cell dataset
sobj_w.3 <- ModuleEigengenes(
  sobj_w.3,
  group.by.vars="cluster.names",
  wgcna_name = "week.3.p"
)

# harmonized module eigengenes:
hMEs <- GetMEs(sobj_w.3)
# module eigengenes:
MEs <- GetMEs(sobj_w.3, harmonized=FALSE)
# compute eigengene-based connectivity (kME):
sobj_w.3 <- ModuleConnectivity(
  sobj_w.3,
  group.by = 'cluster.names', group_name = 'progenitor'
)
# rename the modules
sobj_w.3 <- ResetModuleNames(
  sobj_w.3,
  new_name = "prog_module"
)
```

**R session info**
Here is the error
> sobj_w.3. <- ResetModuleNames(
+   sobj_w.3,
+   new_name = "prog_module"
+ )
Error in GetROCData(seurat_obj, wgcna_name) : 
  could not find function "GetROCData"

**Screenshots**
on mac
![Screenshot 2023-03-29 at 5 05 13 PM](https://user-images.githubusercontent.com/33120647/228667410-b5e470a5-61d2-4e52-8a4d-47148aba7e01.png)
on windows
![roc (1)](https://user-images.githubusercontent.com/33120647/228667531-44f67742-befb-4ac9-b2c4-2091c44fa915.png)

**Comment by smorabit** [comment]
Hi, 

To clarify, those are old functions that are no longer used so I just deleted them, but I must have left some references to them hanging around. I just fixed this so please try to re-install the package and run it again.

**Comment by RcuIT** [comment]
Hi Thanks for the reply. I re-installed hdWGCNA and started again. I skipped the `ResetModuleNames` function as it gave me the same error. Thereafter, I tried to perform, `ModuleNetworkPlot(sobj_w.3)` function and gave me another error saying can't find pdf function. any suggestions are appreciated. 
![Screenshot 2023-03-30 at 10 21 48 AM](https://user-images.githubusercontent.com/33120647/228867091-d0c0330a-1f91-4cc9-9583-2ae9b7fd5fa3.png)
